src/main/java/org/folio/oaipmh/validator/VerbValidator.java

```diff
@@ -80,6 +73,13 @@ private void validateExclusiveParam(Verb verb, Map<String, String> requestParams
           .withCode(BAD_RESUMPTION_TOKEN)
           .withValue(format(INVALID_RESUMPTION_TOKEN, verb.name()));
         errors.add(error);
+        return;
+      }
+      if (!request.isResumptionTokenTimeExpired()) {
+        OAIPMHerrorType errorByExpiredTime = new OAIPMHerrorType()
+          .withCode(BAD_RESUMPTION_TOKEN)
```

Comment:
Seems that logic with error creation is executed if resumption token is not expired.